macros.in

```diff
@@ -542,6 +542,12 @@ Supplements:   (%{name} = %{version}-%{release} and langpacks-%{1})\
 # Use internal dependency generator rather than external helpers?
 %_use_internal_dependency_generator	1
 
+#
+# Generate minimum versions for ELF libraries that don't provide
+# versioned symbol?
+#%_provide_fallback_versions   --libtool-version-fallback
+#%_require_fallback_versions   --libtool-version-fallback
```

Comment:
I think these ought to have elf-prefixes (ie %_elf_provide...), just to be on the safe side. Historically rpm has been terribly sloppy wrt namespaces and we can no longer afford to do that.